ROOMBA_INDEXES: [1, 3, 5]
ROOMBA_COLORS: ["green", "blue", "purple"]
